### ISSUE TITLE
Docker image downloads all but top 3 layers.

### ISSUE THREAD

**Issue by RoninHunter** [dusty-nv/jetson-inference#1480]
All but the top 3 layers download. 

I get a "Pulling fs layer" for the first 3 layers in the list. 

Is this normal or is there something wrong here?

**Comment by dusty-nv** [comment]
Hmm based on your other post, I think you may be having a network issue.  Does it ever finish pulling?   You can press Ctrl+C and try again if it's stuck.

**Comment by RoninHunter** [comment]
Yeah not sure what is going on with this one. I actually tried it after posting with another ubuntu machine that has docker installed and it did the exact same thing. In the end it worked for some unexplainable reason on the original machine.
